### PR TITLE
Sorting out bits and pieces

### PR DESCRIPTION
**Pull request by paulbessell**
@BlairArchibald If you get five minutes could you possibly have a look at this. I don't understand why it is failing these tests but I am not sure they are to do with me. They are to do wiht the quarantine tests that I haven't really touched, it may be all related to the second failed test in that somehting has happenned to the parameters file.

If you don't get a chance thne no worries and I will come back to it later

### CODE REVIEW

**Comment by BlairArchibald** [comment]
That's the tests working now. Please check that what I did to quarantine is sane - I think the original probabilities were upside down (so pQuarantine = 0.9 was only a 10% chance not 90%. I assume w want 90%?)

File: src/main/java/uk/co/ramp/covid/simulation/parameters/HouseholdProperties.java [hunk 1]
```diff
@@ -4,7 +4,7 @@
 
 public class HouseholdProperties {
     public Probability pVisitorsLeaveHousehold = null;
-    public Probability pHouseholdVisitsNeighbourDaily = null;
+    public Double pHouseholdVisitsNeighbourDaily = null;
```
Comment (BlairArchibald) [inline src/main/java/uk/co/ramp/covid/simulation/parameters/HouseholdProperties.java:7]: If this isn't a Probability we should drop the "p" prefix

**Comment by paulbessell** [comment]
> That's the tests working now. Please check that what I did to quarantine is sane - I think the original probabilities were upside down (so pQuarantine = 0.9 was only a 10% chance not 90%. I assume w want 90%?)

Makes perfect sense, populating the probability class with a value of 0.9 is bang on

**Comment by BlairArchibald** [comment]
Can you also remove "quarantineVal" since this isn't needed anymore either

**Comment by paulbessell** [comment]
Final names tweaked. Thanks for fixing that Blair

**Comment by paulbessell** [comment]
> Happy for this to go in as is, and we can remove the quarantineVal later - although ideally we do it now if you have time?
> 
> You possibly need to switch this from a draft @paulbessell before we can merge.

Yes, can you tell me where this parameter is

**Comment by paulbessell** [comment]
> > Happy for this to go in as is, and we can remove the quarantineVal later - although ideally we do it now if you have time?
> > You possibly need to switch this from a draft @paulbessell before we can merge.
> 
> Yes, can you tell me where this parameter is

Sorry, wrote that durin a call. I can't find the parameter anywhere, but I think I am being dumb

**Comment by BlairArchibald** [comment]
Hi @paulbessell  it's in Covid.java ?

**Comment by paulbessell** [comment]
Reight, definitely can't find quarantineVal in Covid.java, so merging what we have

**Pull request merged**